addons/l10n_it_edi/models/ir_mail_server.py

```diff
@@ -359,6 +359,19 @@ def _return_error_xml(self, tree):
 class IrMailServer(models.Model):
     _name = "ir.mail_server"
     _inherit = "ir.mail_server"
+    def _get_test_email_addresses(self):
+        self.ensure_one()
+        company = self.env["res.company"].search([("l10n_it_mail_pec_server_id", "=", self.id)], limit=1)
```

Comment:
Not sure if it is such a good idea to send a mail to the FatturaPA as a test.  (there is no test environment here...)  I wonder if you could send a mail to yourself instead?

Reply:
This should not be a problem because the mail is not really sent, but uses some  tricks of smtp protocol